docs/content/tutorial/scheduling-your-pipeline.mdx

```diff
@@ -0,0 +1,158 @@
+---
+title: "Tutorial, part five: Scheduling your pipeline | Dagster Docs"
+description: Learn how to schedule your pipeline to run at a set frequency.
+---
+
+# Tutorial, part five: Scheduling your pipeline
+
+Now that you've written an entire pipeline in Dagster, you will need to run it regularly to keep it up to date.
+
+By the end of this part of the tutorial, you'll be able to:
+
+- Structure your project with code locations and jobs
+- Refresh your assets periodically with schedules
+- Understand your data's freshness and staleness
```

Comment:
This isn't covered here, right? If so, should we take this bullet out?

Reply:
Ohp, that's an artifact from the outline. I was originally thinking about talking about freshness, but that would probably double the length of this page. Dropping it!